cmd/integration/integration.go

```diff
@@ -192,15 +190,30 @@ func startComponents(manifestURL string) (apiServerURL string) {
 	nodeController.Run(10 * time.Second)
 
 	// Kubelet (localhost)
+	testRootDir := makeTempDirOrDie("kubelet_integ_1.")
```

Comment:
leave them for debugging? I did have the similar question on the original RootDirs.